daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto

```diff
@@ -1487,6 +1487,7 @@ message DefDataType {
     Fields record = 3; // Records without fields are explicitly allowed.
     Fields variant = 4; // Variants without constructors are explicitly allowed.
     EnumConstructors enum = 7;
+    Unit interface = 9; // Interfaces are special, they can't be represented as records or variants.
```

Comment:
I thought we said we don’t add them here for now since they’re redudant?

Reply:
yes, I stumbled again over this and found it really weird to add an error case for the encoding of the DataInterface. Where do we have a guarantee that there is nowhere a DataInterface type left when encoding?

Reply:
alright, I’m fine with keeping this as well.